Std/Data/AssocList.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura, Mario Carneiro
 -/
 import Std.Data.List.Basic
+import Std.Util.ProofWanted
```

Comment:
```suggestion
```
you didn't use it

Reply:
I guess we should import this early somewhere, to avoid having this happen too often?

Reply:
Yes, I think `Tactic.Basic` has a few others like this.